web/src/service-worker/index.ts

```diff
@@ -11,8 +11,15 @@ const pendingLoads = new Map<string, AbortController>();
 // Create a unique cache name for this deployment
 const CACHE = `cache-${version}`;
```

Comment:
This version number is auto-incremented by Svelte everytime there is a new build.